### PR TITLE
Task 3 on Python realization

### LINKED ISSUE #5: Task3_Python
Create script that takes the file path and extension of files as arguments,
and returns count of files with the specified extension in this directory.

### PR DESCRIPTION
**Pull request by dnason**
## Describe your changes
      Initial commit
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [x] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

**Review by Legamanchik — APPROVED** [review]

File: task3.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import os
+import sys
+
+def count_files_by_extension(directory_path, extension):
+    try:
+        count = 0
+        for root, dirs, files in os.walk(directory_path):
```
Comment (arkma) [inline task3.py]:
```suggestion
        for _, _, files in os.walk(directory_path):
```

Comment (arkma) [reply task3.py]: We can ignore unused positional arguments in python using '_' instead. Everything else looks good.

**Review by sbodia — COMMENTED** [review]
It would also be great to add a little check that the path exists, like:
```
if not os.path.isdir(directory):
    # error message
```

**Review by arkma — APPROVED** [review]

File: task3.py (hunk 1 above)
Comment (dnason) [reply task3.py]: Agreed

**Review by AndriyGalias — APPROVED** [review]
Everything looks fine

**Pull request merged**